src/lxc/commands.c

```diff
@@ -835,7 +835,7 @@ static int lxc_cmd_get_name_callback(int fd, struct lxc_cmd_req *req,
 
 	memset(&rsp, 0, sizeof(rsp));
 
-	rsp.data = handler->name;
+	rsp.data = (char *)handler->name;
```

Comment:
How about making rsp.data const void * (as send(2) shows for its data type)?

Reply:
Yeah, I guess that makes sense.

Reply:
Actually no. That won't work since `rsp.data` is also passed to `recv()`.